client/layout/index.jsx

```diff
@@ -116,7 +116,9 @@ class Layout extends Component {
 				) : (
 					<TranslatorLauncher />
 				) }
-				{ this.props.section.group === 'sites' && <SitePreview /> }
+				{ this.props.section.group === 'sites' && (
+					<AsyncLoad require="blocks/site-preview" placeholder={ null } />
+				) }
```

Comment:
This will still load `SitePreview` "almost always", just moving the code away from the critical path of initial render.

Could we add the (hopefully cheap) `getCurrentLayoutFocus( state ) === 'preview'` condition here? That would change the load frequency to "almost never".

Reply:
oooh good call

Reply:
nice, it was already part of the props passed to `Layout`. Added in 03b250fd2b